cases/filters.py

```diff
@@ -14,6 +16,7 @@ def get_wards():
 
 
 class CaseFilter(django_filters.FilterSet):
+    search = django_filters.CharFilter(label="Search", method="search_filter")
```

Comment:
Do you think we could use the [`lbh-search-box` component](https://design-system.hackney.gov.uk/components/search-box) here, instead of a regular text input?

(I was going to try and add it myself, but couldn’t wrap my head around how to define a custom filter type in Django Filters!)

It’ll mean the form has two submit buttons (the maginifying glass icon overlaid on the search box, and the "Filter" button at the end of the form. Can you see any issues with that? Feels quite natural to have a submit button right next to the search box, for super quick searching.

Reply:
Yep, done (you didn't want a custom filter type, just a new widget template):
![image](https://user-images.githubusercontent.com/154364/141513172-b4fdf0bc-47ff-4ca7-9161-af4f64a6e7ba.png)